### DISCUSSION TITLE
Display MCQ Questions layout Issue

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Knowlg/Community#66] [Bugs]
Hi @vinukumar-vs, @gouravmore  
Hope you are doing well!..

**Site Environments Details -** SunbirdEd portal of version 4.6.0

**Issue -** Large-length MCQ questions and their large length of options (answers) are not displayed correctly / in full text PFA.

**CC** @baljeetsingh 

**Attachment 1**. 
![queTemplate1](https://user-images.githubusercontent.com/3881053/189343895-d124521d-fa44-475e-9557-1e26466aa520.png)

**Attachment 2.** 
![queTemplate2](https://user-images.githubusercontent.com/3881053/189343966-13dff107-69fd-4f1d-ad44-229557e628fa.png)

**Attachment 3.**
![queTemplate3](https://user-images.githubusercontent.com/3881053/189343987-c47afe3c-6ebf-4f29-8cc1-12a4e9fbb178.png)

**Attachment 4.** 
![queTemplate4](https://user-images.githubusercontent.com/3881053/189344036-6bbd01ae-15ae-44f7-a4ef-645d2a821e75.png)

**Attachment 5.**
![queTemplate5](https://user-images.githubusercontent.com/3881053/189344074-455d9789-3301-4b90-a866-2a6b09070f86.png)

**Attachment 6.** 
![queTemplate6](https://user-images.githubusercontent.com/3881053/189344130-a460327a-b812-4848-a1f7-feb993dde7a1.png)

**Comment by komalm** [comment]
> @gouravmore

Can you please help us...

**Comment by surendrasinghs** [comment]
A quick explanation... Each MCQ layout is designed for a specific use case, keeping in mind a specific need. See [screenshots in this presentation](https://docs.google.com/presentation/d/1tTJqdeg7IeCtPmdUOQxz6631q6jOEFcgGK1vHPWuVUI/edit?usp=sharing). Some of the layouts might have changed slightly since the time this presentation was created.

Looking at this I think only Attachment 1 is not as expected. The rest of them seem to be as per the design. Can you please check and share your analysis?

**Comment by gouravmore** [comment]
@komalm For attachment 1 - We have already fixed this issue in release-4.8.0. Please find the bellow Patch link for this fix.
Link:- https://github.com/project-sunbird/sunbird-content-plugins/pull/754
